web/ensign-user-ui/public/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Ensign User UI by Rotational Labs</title>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" /> 
+    <meta name="description" content="User UI for Ensign"/>
+
+    <meta property="og:url" content="https://rotational.app">
+    <meta property="og:type" content="website">
+    <meta property="og:title" content="Ensign by Rotational Labs">
+    <meta property="og:description" content="Ensign by Rotational Labs">
+    <meta property="og:image" content="">
```

Comment:
I will create a story to confirm the description and image that should be used with Edwin.